P202-Training.Domain/RoleService.cs

```diff
@@ -0,0 +1,63 @@
+﻿using P202.Training.Data.Entities;
+using P202.Training.Data.Repositories;
+using System.Collections.Generic;
+using P202.Training.Data;
+using AutoMapper;
+
+namespace P202.Training.Domain
+{
+    public class RoleService : IRoleService
+    {
+        private RoleRepository _repository;
+        public void CreateRole(Role role)
+        {
+            if (role == null)
+                return;
+            Mapper.Initialize(cfg => cfg.CreateMap<Role, Role>());
```

Comment:
Same thing is happening in @darkainsoul PR. We shouldn't be using static classes like because they are "hidden dependencies". Instead we should be injecting an `IMapper` and use that instance. If Automapper doesn't provide one, then perhaps you can create a class that wraps the `Mapper` static class